ob_intern/Controller/Short Video List/ShortVideoListViewController.swift

```diff
@@ -52,15 +53,20 @@ class ShortVideoListViewController: UIViewController {
     
     //MARK: - Functions
     func setupView() {
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
         collectionView.delegate = self
         collectionView.dataSource = self
+        let nibName = String(describing: ShortVideoListCollectionViewCell.self)
+        collectionView.register(UINib(nibName: nibName, bundle: nil), forCellWithReuseIdentifier: nibName)
         collectionView.register(UINib(nibName: String(describing: ShortVideoListCollectionViewCell.self), bundle: nil), forCellWithReuseIdentifier: "ShortVideoListCollectionViewCell")
```

Comment:
อันนี้ให้แก้เพราะมันใช้เหมือนกันครับ จะได้ใช้ค่าแบบไม่เสียเที่ยว
```suggestion
        collectionView.register(UINib(nibName: String(describing: ShortVideoListCollectionViewCell.self), bundle: nil), forCellWithReuseIdentifier: nibName)
```